aiohttp/connector.py

```diff
@@ -63,6 +65,23 @@
     SSLContext = object  # type: ignore[misc,assignment]
 
 
+EMPTY_SCHEMA_SET = frozenset({""})
```

Comment:
I'm not really following what the purpose of the empty set is?

Reply:
Its existing. The change it to break it out into its own constant.

I think its to allow connecting to "example.com/x/y/z"